OneSignalExample/Packages/com.onesignal.unity.core/Editor/SettingsWindow/SettingsTab.cs

```diff
@@ -0,0 +1,18 @@
+using Com.OneSignal.Editor.IMGUI;
+using UnityEditor;
+using UnityEngine;
+
+namespace Com.OneSignal.Editor
+{
+    class SettingsTab : WindowTabElement
+    {
+        public override void OnGUI()
+        {
+            var settings = OneSignalSettings.Instance;
+            using (new WindowBlockWithIndent(new GUIContent("Settings")))
+            {
+                settings.ApplicationId = EditorGUILayout.TextField("Application Id: ", settings.ApplicationId);
```

Comment:
This is great but I think we will want to instruct the user on this screen they still need to make sure they initialize OneSignal from code on the first Scene that loads with their app.